test/integration/targets/collections_runtime_pythonpath/runme.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -mvEeuxo pipefail
```

Comment:
Remove the extra options to cut down on the verbosity of the test. This will make the test output much easier to follow by removing unnecessarily verbose output, which is nearly half the output of the test.

```suggestion
set -eux -o pipefail
```

Reply:
That's just `-v`, why do you want to cut off others?